fl4health/clients/basic_client.py

```diff
@@ -779,12 +781,23 @@ def get_model(self, config: Config) -> nn.Module:
         """
         raise NotImplementedError
 
+    def update_before_train(self, current_server_round: int) -> None:
+        """
+        Hook method called before training with the number of current server rounds performed.
+        For example, used by Moon and Fenda model to save global modules after aggregation.
```

Comment:
Did you mean "models" instead?

Reply:
Actually I meant modules for modules as we call them global and local module in Fenda model. But I agree that model should be updated to models.

Reply:
oh ok, thanks for clarifying :)